fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentImplicits.scala

```diff
@@ -0,0 +1,119 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.{Element, Tag, Content}
+import com.gu.facia.api.models.{ImageReplace, ImageCutout, FaciaContent}
+import org.joda.time.DateTime
+
+import scala.util.Try
+
+object FaciaContentImplicits {
+
+  implicit class FaciaContentImplicit(fc: FaciaContent) {
```

Comment:
I use the implicits, it's much cleaner to use. I can keep this in frontend only if you prefer.

If I had to pick, I would pick the implicit way for `FaciaContent`. I have only provided the other way because you said you don't like it @adamnfish.